style.css

```diff
@@ -12,7 +12,7 @@
 
 #nav-bar {
   position: fixed;
-  width: 100%;
+  width: 100vw;
```

Comment:
- [ ] Good job so far your mobile menu works as expected :clap: However there are some issues with your design :cry: Please make sure that your menu option underline is smaller like the one on the Figma template. Please make ensure you that your menu Xmark has the same size, color, and position as defined on the Figma template because as a developer it is important to pay attention to details and stick to the chosen template as much as possible :smile:
Please check the screenshots below  to see the differences :arrow_down: 

#### Figma Design
![Screenshot from 2022-09-14 10-09-31](https://user-images.githubusercontent.com/6615394/190126769-13750d4f-23ba-4953-af17-bc64c40a83b4.png)

#### Your Design
![Screenshot from 2022-09-14 09-51-48](https://user-images.githubusercontent.com/6615394/190126781-9bea12a0-ea25-4fc4-8fee-48dd31f467dd.jpg)

Reply:
Please make sure to fix the issues pointed out by the previous reviewer, the close button is still wronly positioned and the text underline is still thicker than the one on Figma:

## Your design:
![image](https://user-images.githubusercontent.com/50721479/190220767-49be7008-6653-47f0-8e5f-6b844f709314.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/190220881-497fb855-267d-4d0e-ad8c-f9ba34ad26bf.png)